### DISCUSSION TITLE
3.1.13

### DISCUSSION THREAD

**Discussion by sirjuddington** [sirjuddington/SLADE#1256] [Releases]
**General**  
- Updated ZDoom, MAPINFO and ACS language definitions (thanks @Blue-Shadow)
- Updated SRB2 Lua language definition (thanks @SteelTitanium)
- Added extra UDMF flags (thanks @Gaerzi)

**Texture Editor**
- Fixed an issue with the 'optional' keyword in TEXTURES

**Map Editor**
- Various improvements and fixes to texture handling and precedence (thanks @Talon1024)

<hr /><em>This discussion was created from the release <a href='https://github.com/sirjuddington/SLADE/releases/tag/3.1.13'>3.1.13</a>.</em>

**Comment by juney2008** [comment]
cool! we got an update update after a long time whiting! also i hope you fix the mac build soon!
